<launch>
  <arg name="station_id" default="0"/>
    <node pkg="swarm_bridge" name="station_bridge_node_tcp" type="tcp_station" output="screen">

      <param name="self_id" value="$(arg station_id)"/>
      <param name="is_ground_station" value="true"/>
      <param name="drone_num" value="10"/>
      <param name="drone_ip_0" value="192.168.1.151" type="string"/>
      <param name="drone_ip_1" value="192.168.1.23" type="string"/>
      <param name="drone_ip_2" value="192.168.1.21" type="string"/>
      <param name="drone_ip_3" value="192.168.1.22" type="string"/>
      <param name="drone_ip_4" value="192.168.1.25" type="string"/>
      <param name="drone_ip_5" value="192.168.1.154" type="string"/>
      <param name="drone_ip_6" value="192.168.1.27" type="string"/>
      <param name="drone_ip_7" value="192.168.1.28" type="string"/>
      <param name="drone_ip_8" value="192.168.1.35" type="string"/>
      <param name="drone_ip_9" value="192.168.1.36" type="string"/>

      <param name="ground_station_num" value="1"/>
      <param name="ground_station_ip_0" value="192.168.1.200" type="string"/>
    </node>

</launch>
